libvips/deprecated/im_gradcor.c

```diff
@@ -47,80 +46,89 @@
+YGRAD_GEN_DECLARATION(gint16);
+XGRAD_GEN_DECLARATION(guint32);
+YGRAD_GEN_DECLARATION(guint32);
+XGRAD_GEN_DECLARATION(gint32);
+YGRAD_GEN_DECLARATION(gint32);
 #if 0
 XGRAD_GEN_DECLARATION( float );
 YGRAD_GEN_DECLARATION( float );
 XGRAD_GEN_DECLARATION( double );
 YGRAD_GEN_DECLARATION( double );
 #endif
 
-int im_gradcor_raw( IMAGE *large, IMAGE *small, IMAGE *out ){
+int
+im_gradcor_raw(IMAGE *large, IMAGE *small, IMAGE *out)
+{
 #define FUNCTION_NAME "im_gradcor_raw"
 
-  if( im_piocheck( large, out ) || im_pincheck( small ) )
-    return -1;
-
-  if( im_check_uncoded( "im_gradcor", large ) ||
-	im_check_mono( "im_gradcor", large ) || 
-	im_check_uncoded( "im_gradcor", small ) ||
-	im_check_mono( "im_gradcor", small ) || 
-	im_check_format_same( "im_gradcor", large, small ) ||
-	im_check_int( "im_gradcor", large ) )
-	return( -1 );
-
-  if( large-> Xsize < small-> Xsize || large-> Ysize < small-> Ysize ){
-    im_error( FUNCTION_NAME, "second image must be smaller than first" );
-    return -1;
-  }
-  if( im_cp_desc( out, large ) )
-    return -1;
-
-  out-> Xsize= 1 + large-> Xsize - small-> Xsize;
-  out-> Ysize= 1 + large-> Ysize - small-> Ysize;
-  out-> BandFmt= IM_BANDFMT_FLOAT;
-
-  if( im_demand_hint( out, IM_FATSTRIP, large, NULL ) )
-    return -1;
-
-  {
-    IMAGE *xgrad= im_open_local( out, FUNCTION_NAME ": xgrad", "t" );
-    IMAGE *ygrad= im_open_local( out, FUNCTION_NAME ": ygrad", "t" );
-    IMAGE **grads= im_allocate_input_array( out, xgrad, ygrad, NULL );
-
-    return ! xgrad || ! ygrad || ! grads
-      || im_grad_x( small, xgrad )
-      || im_grad_y( small, ygrad )
-      || im_generate( out, gradcor_start, gradcor_gen, gradcor_stop, (void*) large, (void*) grads );
-  }
+	if (im_piocheck(large, out) || im_pincheck(small))
+		return (-1);
+
+	if (im_check_uncoded("im_gradcor", large) ||
+		im_check_mono("im_gradcor", large) ||
+		im_check_uncoded("im_gradcor", small) ||
+		im_check_mono("im_gradcor", small) ||
+		im_check_format_same("im_gradcor", large, small) ||
+		im_check_int("im_gradcor", large))
+		return (-1);
+
+	if (large->Xsize < small->Xsize ||
+		large->Ysize < small->Ysize) {
+		im_error(FUNCTION_NAME, "second image must be smaller than first");
+		return (-1);
+	}
+	if (im_cp_desc(out, large))
+		return (-1);
+
+	out->Xsize = 1 + large->Xsize - small->Xsize;
+	out->Ysize = 1 + large->Ysize - small->Ysize;
+	out->BandFmt = IM_BANDFMT_FLOAT;
+
+	if (im_demand_hint(out, IM_FATSTRIP, large, NULL))
+		return (-1);
+
+	{
+		IMAGE *xgrad = im_open_local(out, FUNCTION_NAME ": xgrad", "t");
+		IMAGE *ygrad = im_open_local(out, FUNCTION_NAME ": ygrad", "t");
+		IMAGE **grads = im_allocate_input_array(out, xgrad, ygrad, NULL);
+
+		if (!xgrad ||
+			!ygrad ||
+			!grads ||
+			im_grad_x(small, xgrad) ||
+			im_grad_y(small, ygrad))
+			return (-1);
+
+		if (im_generate(out,
+				gradcor_start, gradcor_gen, gradcor_stop, (void *) large, (void *) grads))
+			return (-1);
+
+		return (0);
```

Comment:
Split into two if-statements, otherwise clang-format placed it in one long line.

I also manually added parentheses to the return statements in this file, but maybe I should do that in all C files.